etc/docker-scripts/bootstrapper.py

```diff
@@ -13,19 +13,42 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-import shutil
 import subprocess
 import sys
+from packaging import version
 
 
-def import_with_auto_install(package):
-    try:
-        return __import__(package)
-    except ImportError:
-        print('Updating package {}'.format(package))
-        subprocess.check_call([sys.executable, '-m', 'pip', 'install', '--upgrade', package])
+def package_install():
+
+    elyra_package_list = package_list_to_dict("requirements-elyra.txt")
+    current_package_list = package_list_to_dict("requirements-current.txt")
+    to_install_list = []
+
+    for package, ver in elyra_package_list.items():
+        if package in current_package_list.keys():
```

Comment:
I don't know how efficient python is, but we might want to set a variable to the current_package_list keys outside the for loop and just reference that list in the `if` statement.  Or, I think `if package in current_package_list:` will work as well.  Then we're not generating an array of keys on each iteration.  I ran across this SO hit: https://stackoverflow.com/questions/4730993/python-key-in-dict-keys-performance-for-large-dictionaries.